src/app/app-components/courses/courses-list/courses-list.component.html

```diff
@@ -1,8 +1,14 @@
-<div *ngFor="let course of courses">
-    <app-courses-item [courseInfo]="course" (deleteCourse)="onDeleteItem($event)">
-    </app-courses-item>
-</div>
+<ng-container *ngIf="courses && courses.length > 0">
+    <div *ngFor="let course of courses">
+        <app-courses-item [courseInfo]="course" (deleteCourse)="onDeleteItem($event)">
+        </app-courses-item>
+    </div>
 
-<div class="rectangle" (click)="onLoadMore()">
-    <span>LOAD MORE</span>
+    <div class="rectangle" (click)="onLoadMore()">
+        <span>LOAD MORE</span>
+    </div>
+</ng-container>
+
+<div *ngIf="!courses || courses.length === 0" class="rectangle">
```

Comment:
Pro tip: Use `Array.isArray(courses)` to check if a variable holds an object of `Array` type.

Reply:
Awesome, I'll start using this validation.

Thanks for the pro-tip!